### ISSUE TITLE
Social Media Plan

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/hq#244]
Links to #201 

[Facebook](https://www.facebook.com/dwylhq)
[Instagram](https://www.instagram.com/dwylhq/)
[Twitter](https://twitter.com/dwyl)

- Dodgeball photos / gifs

**Comment by markwilliamfirth** [comment]
Waiting on Instagram + Facebook username claims

**Comment by iteles** [comment]
@markwilliamfirth This looks like a great event to attend and write a blog post on: https://generalassemb.ly/education/mindful-digital-marketing/london/37521

Thoughts?

**Comment by markwilliamfirth** [comment]
Linked to #201

**Comment by iteles** [comment]
https://twitter.com/realjefflarge/status/961330248280309760

**Comment by rub1e** [comment]
@iteles I'll close this later for a new & more explicit / clearer issue with tasks and whatnot

(We can discuss the tweet you've linked there?)

Before I do, can you let me know where the write-up of the General Assembly event is kept (referred to in https://github.com/dwyl/hq/issues/244#issuecomment-307622371). I looked on the team drive but it's in such a mess I couldn't tell if this hadn't been done at all, or if it had and was just misleadingly named 🤔

**Comment by iteles** [comment]
@rub1e Mark didn't go to the event (or acknowledge the comment) so there's no write-up.

Thanks, a more explicit issue would be fantastic!

**Comment by rub1e** [comment]
Closing in favour of #431
